content/en/platform/corda/4.8/open-source/key-concepts-ecosystem.md

```diff
@@ -12,59 +12,40 @@ menu:
 tags:
 - concepts
 - ecosystem
-title: Network
+- network
+title: Networks
 ---
 
 
-# Network
+# Networks
 
 ## Summary
 
-* *A Corda network is made up of nodes, each of which runs an instance of Corda and one or more CorDapps*
-* *Communication between nodes is point-to-point, and does not rely on global broadcasts*
-* *Each node has a certificate that maps its network identity to a real-world legal identity*
-* *The network is permissioned, with access requiring a certificate from the network operator*
+* *People and businesses using Corda can communicate over a peer-to-peer network.*
+* *A Corda network is made up of nodes, which represent real-world legal identities. Each of which runs an instance of Corda and one or more CorDapps.*
+* *Nodes communicate point-to-point—not by global broadcast.*
+* *Each node has a certificate that maps its network identity to a real-world legal identity.*
+* *Corda networks are semi-private—you need a certificate from the network operator to join.*
 
-## Network structure
-
-A Corda network is a peer-to-peer network of **nodes**. Each node represents a legal entity, and each runs the Corda software (an instance of Corda and one or more Corda applications, known as **CorDapps**).
+## What is a network?
+On Corda, people and business interact by communicating over a peer-to-peer network of [Corda nodes](key-concepts-node.md). Each node represents a legal entity running Corda and one or more Corda distributed applications, known as **CorDapps**.
 
 {{< figure alt="network" width=80% zoom="/en/images/network.png" >}}
-All communication between nodes is point-to-point and encrypted using transport-layer security. This means that data is
-shared only on a need-to-know basis. There are **no global broadcasts**.
-
-All of the nodes in the network have the *potential* to communicate with other nodes.
-
-**Why do we say “potential” to communicate?**
+Corda is different from other distributed ledgers because all communication between nodes is point-to-point, and only shared on a need-to-know basis. It's also encrypted using transport-layer security. There are **no global broadcasts** to all parties on a network, but all of the nodes in a network can send messages directly to other nodes. If the recipient is offline, the message waits in an outbound queue until they are online again—just like email.
```

Comment:
```suggestion
Corda is different from other distributed ledgers because all communication between nodes is point-to-point, and only shared on a need-to-know basis. It's also encrypted using transport-layer security. There are **no global broadcasts** to all parties on a network, but all of the nodes in a network can send messages directly to other nodes. If the recipient is offline, the message waits in an outbound queue until they are online again — just like email.
```
same comment as above